src/command/dev/watcher.rs

```diff
@@ -34,29 +35,38 @@ impl SubgraphSchemaWatcher {
             schema_watcher_kind: SubgraphSchemaWatcherKind::File(path.as_ref().to_path_buf()),
             subgraph_key,
             message_sender,
+            polling_interval: 1,
```

Comment:
one more thing: let's nest this `polling_interval` in the `SubgraphSchemaWatcherKind` type so that you don't have to set a value that won't be used for the file watcher.

Reply:
I've implement this inside SubgraphSchemaWatcherKind here: https://github.com/apollographql/rover/pull/1377/commits/00eda0e99742b97aee6bb8b8b276faad65619e2d

I was trying to implement a top level `IntrospectRunner` but I didn't find a solution that I liked, mostly because of naming (we already have GraphIntrospectRunner for example). I think we could do this in another PR 😊